static/script/devices/mediaplayer/html5.js

```diff
@@ -124,6 +124,7 @@ require.def(
                         this._toBuffering();
                         this._targetSeekTime = this._getClampedTimeForPlayFrom(seconds);
                         if (this._isNearToCurrentTime(this._targetSeekTime)) {
+                            this._targetSeekTime = undefined;
```

Comment:
We've worked our way through the case that gave rise to this, and we can see the unit test that drove it out, so we are happy that this is both a necessary fix and a general improvement :-)